src/config/authd-config.c

```diff
@@ -119,15 +120,21 @@ int Read_Authd(const OS_XML *xml, XML_NODE node, void *d1, __attribute__((unused
 
             config->flags.use_source_ip = b;
         } else if (!strcmp(node[i]->element, xml_force_insert)) {
-            mdebug1("The <%s> tag is deprecated since version 4.3.0.", xml_force_insert);
+            mwarn("The <%s> tag is deprecated since version 4.3.0. Use <%s> tag instead.", xml_force_insert, xml_force);
             short b = eval_bool(node[i]->content);
             if (b < 0) {
                 merror(XML_VALUEERR, node[i]->element, node[i]->content);
                 return OS_INVALID;
             }
             legacy_force_insert = b;
         } else if (!strcmp(node[i]->element, xml_force_time)) {
-             mdebug1("The <%s> tag is deprecated since version 4.3.0.", xml_force_time);
+            mwarn("The <%s> tag is deprecated since version 4.3.0. Use <%s> instead.", xml_force_time, xml_force);
+            short b = eval_bool(node[i]->content);
```

Comment:
`force_time` value is an integer so we cannot evaluate it as a boolean. Instead of calling `eval_bool` we have to store its value in a `int` variable.

See how in 4.2 the old `force_time` value was read with `strtol`.